### PR TITLE
Current Document Library evaluation and updates to ensure it all works

### PR DESCRIPTION
**Pull request by draperd**
Just spent a bit of time working through the current Document Library to make sure we hadn't regressed anything and took the opportunity to tidy a few things up.

### CODE REVIEW

**Comment by draperd** [comment]
Don't pull this yet - I need to push a couple of updates to unit tests that are failing.

**Comment by draperd** [comment]
Pushed the required test updates

File: aikau/src/main/resources/alfresco/documentlibrary/AlfBreadcrumb.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ define(["dojo/_base/declare",
       postCreate: function alfresco_documentlibrary_AlfBreadcrumbTrail__postCreate() {
```
Comment (davidcognite) [inline aikau/src/main/resources/alfresco/documentlibrary/AlfBreadcrumb.js]: Name shouldn't contain "Trail"

File: aikau/src/main/resources/alfresco/documentlibrary/AlfBreadcrumb.js [hunk 2]
```diff
@@ -102,7 +102,7 @@ define(["dojo/_base/declare",
...
          if (this.label)
          {
-            this.breadcrumbNode.innerHTML = this.encodeHTML(this.message(this.label));
+            this.breadcrumbNode.innerHTML = this.message(this.label);
```
Comment (davidcognite) [inline aikau/src/main/resources/alfresco/documentlibrary/AlfBreadcrumb.js]: Is the label sanitised elsewhere?

Comment (draperd) [inline aikau/src/main/resources/alfresco/documentlibrary/AlfBreadcrumb.js]: The filter publications are currently only issues by filters that have to be manually added to page, so there is minimal rish. The issue here was that when clicking on any of the tag filters we were getting double encoding of quotes (which is why I've removed the encoding).

**Pull request merged**